test/blocks/uar-results/uar-results.test.js

```diff
@@ -0,0 +1,66 @@
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+import { stub } from 'sinon';
+import { delay } from '../../helpers/waitfor.js';
+
+window.lana = { log: stub() };
+localStorage.clear();
+
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+const { default: init } = await import('../../../libs/blocks/uar-results/uar-results.js');
+const { default: mockData } = await import('./mocks/uar-results.mock-data.js');
```

Comment:
If you put these three lines in a `before` block it might help with the race condition issues causing the need for `delay` below. We've been finding readFile is finicky and can cause flaky test results.

Reply:
The delay is present to ensure that the fragment loading has happened, before checking that the block has loaded - it's not really about a race condition for the files to be loeded. The first two test that don't get to loading a fragment don't require the delay.